cloud_topic: /fastlio2/body_cloud

odom_topic: /fastlio2/lio_odom

map_frame: map

local_frame: lidar

key_pose_delta_deg: 20.0               # 关键帧角度变化阈值（单位：度），大场景增大角度阈值减少关键帧密度

key_pose_delta_trans: 3.0           # 关键帧平移变化阈值（单位：米），大场景地下车库增大距离阈值

loop_search_radius: 15.0            # 回环检测时的搜索半径（单位：米），大场景地下车库增大搜索范围

loop_time_tresh: 15.0                # 回环检测的时间间隔阈值（单位：秒），大场景增大时间间隔避免过密检测

loop_score_tresh: 0.2               # 回环匹配得分阈值（越小匹配越好），大场景稍微放宽阈值

loop_submap_half_range: 35           # 回环子图半径（单位：米），大场景增大子图范围

submap_resolution: 0.2              # 子图匹配时的体素滤波分辨率（单位：米），大场景可适当增大

min_loop_detect_duration: 3.0       # 每次回环检测中允许的最小处理时间（单位：秒），大场景增大间隔

loop_m_key_poses: 12                 # 回环检测时，关键帧数量阈值（大场景需要更多关键帧来检测回环）